### DISCUSSION TITLE
有没有doubao 豆包的？

### DISCUSSION THREAD

**Discussion by joostshao** [LLM-Red-Team/.github#6] [Ideas]
https://www.doubao.com/chat/2337086210
有没有doubao 豆包的？

**Comment by Vinlic** [comment]
豆包在投票名单的下一位，不过现在正在进行SmartChatbot计划，会延后。

**Reply by ichat001** [reply]
大佬好, 有逆Coze国际版的计划吗? 
1.我现在正在用这个 https://github.com/deanxv/coze-discord-proxy 项目, 中转了好几次, 速度有点慢, coze机器人功能可用, 但仍有50-500次的限制
2.用官方的扣子API调用, 只能调用100次, 而且调用代码和chatgpt没有对齐, 我用的这个项目https://github.com/fatwang2/coze2openai 对齐的
3.另外, 我发现在Coze机器人的编排模式下(如下图), 使用右侧的预览调试面板, 始终是可以使用chatgpt的, 没有任何次数或功能限制,  那把这个地方逆了不就万事大吉了?
![coze](https://github.com/LLM-Red-Team/.github/assets/166457982/8aeca6f4-3607-4d6b-9aab-9db478af9405)
